src/vasp.jl

```diff
@@ -165,4 +165,59 @@ end
 
 readDOSCAR(filename::AbstractString) = open(readDOSCAR, filename)
 
-readDOSCAR() = open(readDOSCAR, "DOSCAR")
\ No newline at end of file
+readDOSCAR() = open(readDOSCAR, "DOSCAR")
+
+function readPROCAR(io::IO)
+    ln = readline(io)
+    
+    if cmp(split(ln)[2], "lm") == -1
+        error("Not a lm-decomposed PROCAR. Use a different PROCAR.")
+    end
+    
+    # Get number of kpoints, number of bands, number of ions
+    ln = readline(io)
+    nkpt = parse(Int,split(ln)[4])
+    nband = parse(Int,split(ln)[8])
+    nion = parse(Int,split(ln)[12])
```

Comment:
This is fine. Here's another way of doing it, but I don't know if this is any better - this is just for demonstration:
```julia
(nkpt, nband, nion) = parse.(Int, split(readline(io))[[4, 8, 12]])
```
Key takeaway: `parse.(T, v)` applies `parse(T, s)` to an entire vector (you can post a function name with a dot to vectorize it).